Description: revert commit #b6a05a8
 We revert the commit "Fix wrongly generated endpoints when override contains
 path" since it strips out the /wayf part in the Location in
 md:SingleLogoutService and md:SingleSignOnService in the IdP metadata which is
 used by the Delegation Service.
 See https://github.com/simplesamlphp/simplesamlphp/issues/1922
 https://github.com/simplesamlphp/simplesamlphp/commit/b6a05a831accd4fbe62c22ff7d3c41c319f761b2
--- a/src/SimpleSAML/Metadata/MetaDataStorageHandler.php	2023-11-29 17:51:18.000000000 +0100
+++ b/src/SimpleSAML/Metadata/MetaDataStorageHandler.php	2023-12-28 22:26:38.000000000 +0100
@@ -105,7 +105,8 @@
         $httpUtils = new Utils\HTTP();
         $baseurl = $httpUtils->getSelfURLHost() . $config->getBasePath();
         if ($overrideHost !== null) {
-            $baseurl = str_replace('://' . $httpUtils->getSelfHostWithPath() . '/', '://' . $overrideHost . '/', $baseurl);
+            $baseurl = str_replace('://' . $httpUtils->getSelfHost(), '://' . $overrideHost, $baseurl);
+//          $baseurl = str_replace('://' . $httpUtils->getSelfHostWithPath() . '/', '://' . $overrideHost . '/', $baseurl);
         }
 
         if ($set == 'saml20-sp-hosted') {
